app/scripts/controllers/preferences-controller.ts

```diff
@@ -565,20 +666,24 @@ export default class PreferencesController {
    *
    * @deprecated We no longer support specifying a ledger transport type other
    * than webhid, therefore managing a preference is no longer necessary.
-   * @param {LedgerTransportTypes.webhid} ledgerTransportType - 'webhid'
-   * @returns {string} The transport type that was set.
+   * @param ledgerTransportType - 'webhid'
+   * @returns The transport type that was set.
    */
-  setLedgerTransportPreference(ledgerTransportType) {
+  setLedgerTransportPreference(
+    ledgerTransportType: LedgerTransportTypes,
+  ): string {
     this.store.updateState({ ledgerTransportType });
     return ledgerTransportType;
   }
 
   /**
    * A setter for the user preference to dismiss the seed phrase backup reminder
    *
-   * @param {bool} dismissSeedBackUpReminder - User preference for dismissing the back up reminder.
+   * @param dismissSeedBackUpReminder - User preference for dismissing the back up reminder.
    */
-  async setDismissSeedBackUpReminder(dismissSeedBackUpReminder) {
+  async setDismissSeedBackUpReminder(
```

Comment:
I believe this function shouldn't be async in first place, once removed the return type would be void
also this.store.updateState is not a promise no need to await